go/cmd/betabot/main.go

```diff
@@ -206,26 +225,25 @@ func betabot() error {
 					return
 				}
 
-				switch {
-				case gme.Event.Type == bertymessenger.StreamEvent_TypeListEnd:
-					bot.logger.Info("finished replaying logs from the previous sessions", zap.Int("count", handledEvents))
-					bot.isReplaying = false
-				case !bot.isReplaying:
-					// replay is done, let's handle events normally
-					wg.Add(1)
-					go func() {
-						defer wg.Done()
-						err := bot.handleEvent(ctx, gme)
-						if err != nil {
-							bot.logger.Error("handleEvent failed", zap.Error(err))
-						}
-					}()
-				default:
-					// bot.logger.Debug("ignoring already handled event", zap.Any("event", gme))
+				if bot.isReplaying {
+					if gme.Event.Type == bertymessenger.StreamEvent_TypeListEnd {
+						bot.logger.Info("finished replaying logs from the previous sessions", zap.Uint("count", handledEvents))
+						bot.isReplaying = false
+					}
 					// replayed events
-					// noop
+					// bot.logger.Debug("ignoring already handled event", zap.Any("event", gme))
+					handledEvents++
```

Comment:
missing handledEvents if not in replay

Reply:
> missing handledEvents if not in replay

@moul 
Why ? The only time it's red is in :
```go
bot.logger.Info("finished replaying logs from the previous sessions", zap.Uint("count", handledEvents))
```
When quitting replay mode.

Reply:
yes, I forgot to log it elsewhere in the func but my goal was to always log this number